fhir-persistence-jdbc/src/test/java/com/ibm/fhir/persistence/jdbc/test/connection/FHIRUserTransactionAdapterTest.java

```diff
@@ -0,0 +1,217 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.persistence.jdbc.test.connection;
+
+import static org.testng.Assert.assertEquals;
+
+import javax.transaction.Status;
+
+import org.testng.annotations.Test;
+
+import com.ibm.fhir.persistence.jdbc.connection.FHIRUserTransactionAdapter;
+
+/**
+ * Unit test for the {@link FHIRUserTransactionAdapter}
+ */
+public class FHIRUserTransactionAdapterTest {
```

Comment:
Love having this unit test

Reply:
Yeah, this really helps give confidence that the transaction adapter is doing the right thing with all the nesting that happens in our persistence layer.